src/frontend/src/instance.rs

```diff
@@ -275,6 +276,17 @@ impl Instance {
             .context(AlterTableSnafu)
     }
 
+    /// Handle explain expr
+    pub async fn handle_explain(&self, sql: &str, explain_stmt: Explain) -> Result<Output> {
+        if let Some(dist_instance) = &self.dist_instance {
+            dist_instance
+                .handle_sql(sql, Statement::Explain(explain_stmt))
+                .await
+        } else {
+            Ok(Output::AffectedRows(0))
```

Comment:
Here we should also handle explain statement in standalone mode. To implement this, you should create a variant of `ExplainExpr`  in gRPC proto file and implement `client::database::Database::explain` by invoking gRPC service in datanode, just like `handle_select` in standalone mode:
https://github.com/francis-du/greptimedb/blob/91e71169be26f20239d3810188852d040a783681/src/frontend/src/instance.rs#L214-L228

Don't panic. I will implement the gRPC stuff so that you can proceed. Hopefully we will write a doc about the current design of frontend and datanode.